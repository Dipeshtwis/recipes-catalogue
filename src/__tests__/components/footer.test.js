/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import footer from '../../components/Footer';

it('renders footer component correctly', () => {
  const footer = renderer.create(<footer />).toJSON();
  expect(footer).toMatchSnapshot();
});
