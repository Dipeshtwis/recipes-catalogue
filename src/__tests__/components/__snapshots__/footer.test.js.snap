// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component correctly 1`] = `<footer />`;
